packages/contracts/test/verifier.test.ts

```diff
@@ -47,6 +48,197 @@ const signupUser = async (id, unirepContract, attesterId, account) => {
     return { leaf: publicSignals[1], index: leafIndex.toNumber(), epoch }
 }
 
+describe('Epoch key lite proof verifier', function () {
+    this.timeout(300000)
+    let unirepContract
+
+    before(async () => {
+        const accounts = await ethers.getSigners()
+        unirepContract = await deployUnirep(accounts[0])
+
+        const attester = accounts[1]
+        await unirepContract
+            .connect(attester)
+            .attesterSignUp(EPOCH_LENGTH)
+            .then((t) => t.wait())
+    })
+
```

Comment:
can use snapshot to revert evm like #243

Reply:
Ah the verifier tests are mostly stateless. I'm not opposed to adding snapshot/revert logic, but i'd rather not block this pr and #239. Need these for `2.0.0-alpha-3` and zkhack next week.

Reply:
I am also good to go without snapshot logic
or after this is merged I can change it in #239

Reply:
Ok, there shouldn't be any conflicts. It should be possible to copy/paste the hooks into this `describe` block.